### ISSUE TITLE
"open with" confirmation on Android after upgrade version

### ISSUE THREAD

**Issue by malgorzata-kozera** [auth0/react-native-auth0#531]
### Describe the problem

I have upgraded react-native-auth0 from version 2.12.0 to v2.14.0.
After the upgrade when logging in or logout from the app, the pop-up with the "open with" message appears. The app request confirmation on which app to use to continue (it shows two of the same app icons). There is only one app installed on the device.
It happens on Android.

I followed all recommendations mentioned in these cases:
https://community.auth0.com/t/react-native-auth0-open-with-duplicate-after-upgrade-to-2-9-0/68033
and 
https://github.com/auth0/react-native-auth0/issues/519 
Also, I have only one intent filter in `AndroidManifest.xml` and I removed ` android:launchMode="singleTask"` but it didn't help.

Any solutions for that?

### What was the expected behavior?

- Pop-up for android not shows

### Environment
- react-native-auth0 v2.14.0
- react-native version 0.70.2

**Comment by poovamraj** [comment]
Hi @malgorzata-kozera Thanks a lot for raising this. You are right and we had to fix this on our end in the latest version of our SDK.

This is because we added our Auth0.Android library to provide Credentials Manager support. This added libraries that receive intent for Web Authentication into the RNA library which is why there are 2 intent receivers.

In this [PR](https://github.com/auth0/react-native-auth0/pull/533) we have removed the additional ones from Android which should fix this issue.

Thanks again for raising this 🙌
